ja/documentation/quickstart/3/index.md

```diff
@@ -0,0 +1,231 @@
+---
+layout: page
+title: "20分ではじめるRuby"
+lang: ja
+
+header: |
+  <div class="multi-page">
+    <a href="../" title="Part 1">1</a>
+    <span class="separator"> | </span>
+    <a href="../2/" title="Part 2">2</a>
+    <span class="separator"> | </span>
+    <strong>3</strong>
+    <span class="separator"> | </span>
+    <a href="../4/" title="Part 4">4</a>
+  </div>
+  <h1>20分ではじめるRuby</h1>
+
+---
+
+それではgreeterオブジェクトを作り、使ってみましょう。
+
+{% highlight irb %}
+irb(main):035:0> greeter = Greeter.new("Pat")
+=> #<Greeter:0x16cac @name="Pat">
+irb(main):036:0> greeter.say_hi
+Hi Pat!
+=> nil
+irb(main):037:0> greeter.say_bye
+Bye Pat, come back soon.
+=> nil
+{% endhighlight %}
+
+一度`greeter` オブジェクトを作ると、Patという名前をおぼえてくれます。
+ふむ、名前を直接取得しようとするとどうなるでしょう？
+
+{% highlight irb %}
+irb(main):038:0> greeter.@name
+SyntaxError: compile error
+(irb):52: syntax error
+        from (irb):52
+{% endhighlight %}
+
+おっと、これはできませんでした。
+
+## Objectの殻の中
+
+インスタンス変数はオブジェクトの内側に隠されています。とはいえ完全に
+見えないようになっているのではなく、オブジェクトをinspectすれば
+いつでも見ることはできますし、他にもアクセスする手段はあるのですが、
+Rubyはデータを隠しておく、適切なオブジェクト指向アプローチをとっています。
+
+では、Greeterオブジェクトにはどんなメソッドがあるのでしょうか？
+
+{% highlight irb %}
+irb(main):039:0> Greeter.instance_methods
+=> ["method", "send", "object_id", "singleton_methods",
+    "__send__", "equal?", "taint", "frozen?",
+    "instance_variable_get", "kind_of?", "to_a",
+    "instance_eval", "type", "protected_methods", "extend",
+    "eql?", "display", "instance_variable_set", "hash",
+    "is_a?", "to_s", "class", "tainted?", "private_methods",
+    "untaint", "say_hi", "id", "inspect", "==", "===",
+    "clone", "public_methods", "respond_to?", "freeze",
+    "say_bye", "__id__", "=~", "methods", "nil?", "dup",
+    "instance_variables", "instance_of?"]
+{% endhighlight %}
```

Comment:
`instance_methods` returns array of Symbols instead of array of Strings.
and `irb(main):040:0> Greeter.instance_methods(false)` too.